src/lib/dnssd/minimal_mdns/Server.h

```diff
@@ -110,6 +128,13 @@ class ServerBase
     virtual CHIP_ERROR DirectSend(chip::System::PacketBufferHandle && data, const chip::Inet::IPAddress & addr, uint16_t port,
                                   chip::Inet::InterfaceId interface);
 
+    /// Send out a broadcast query. Will use an ephemeral port to receive replies
```

Comment:
```suggestion
    /// Send out a legacy unicast query. Will use an ephemeral port to send queries and receive unicast replies
```

Reply:
Updated comments but  said 'may' for the legacy unicast because  I plan to do a followup where using unicast is configurable.